src/constants.py

```diff
@@ -26,13 +26,13 @@
 # [M, T, W, Th, F, St, S]
 # [0, 1, 2, 3, 4, 5, 6]
 WEEKDAYS = {
-    'monday-wednesday': [0, 1, 2],
-    'monday-thursday': [0, 1, 2, 3],
-    'monday-friday': [0, 1, 2, 3, 4],
-    'thursday': [3],
-    'friday': [4],
-    'saturday': [5],
-    'sunday': [6]
+    'monday': 0,
```

Comment:
how come this got changed from monday-wednesday to monday?

Reply:
the previous method of having 'monday-...' correspond to a list of indexes would require an exhaustive list of day combinations to include all edge cases, having the days correspond to an index and then make the list in a method is more streamlined approach.